### DISCUSSION TITLE
Change mobile viewport size

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#129] [1. Help]
Hi,

How can we change the mobile viewport size to 767px. Currently It's changing to mobile view from 640px. We want to change it to 767px. So mobile header can be viewed till 767px.

Is their any easy way to do it?

**Comment by vovayatsyuk** [comment]
Try to change `@sm` variable:

```less
@sm: 768px;
```

Source: https://breezefront.com/docs/media-queries#responsive-breakpoints
